### ISSUE TITLE
Control Meeting-Location for Users in Team Events

### ISSUE THREAD

**Issue by its-julian** [calcom/cal.com#11652]
### Is your proposal related to a problem?

Currently, Admin's have no control over meeting locations for team events & there is no way to force/regulate this from Admin account in Cal.com. This is a problematic because in our company, member's meetings must be held through Zoom, so that we can record, store, and monitor call-recordings directly from our Admin accounts (this isn't possible with other tools like Daily, or GoogleMeet recordings).

Cal.com does not give us this option. Cal only gives us the option to allow "user default location" which allows our members to use whatever they want, leaving too much room for errors & headache when setting up events & onboarding new members.

### Describe the solution you'd like

We would simply like to have the option to regulate the Meeting Location on a team events. This could be done by adding Zoom/GoogleMeet as an option for "Default Location", or by adding a Dropdown next to each user in the event, where we can select the location for each individual user added to the event.

This is how Calendly does it, and if any user doesn't have Zoom connected when selecting the app for each user, you will see an "app not connected" error, so you can reach out to this member & tell them they need to connect their Zoom account. This gives us much more control over our events, meeting locations, and recordings for our company/team.

**Comment by ErrolFernandes** [comment]
To summarize you want to setup custom meeting app for each user and a default one, correct?

**Comment by its-julian** [comment]
To summarize, I would like to be able to select Zoom/Google Meet as the member's meeting location from the Admin side of the Team Events.

As you can see in the screenshots below. Cal.com only gives you the option to use "organizer's default app", whereas Calendly allows you to manually select the location for each member yourself. Giving us Admin's more control to regulate meeting locations of our teams.

![CleanShot 2023-10-03 at 12 34 47@2x](https://github.com/calcom/cal.com/assets/97125783/45d1c821-fc11-4978-939e-451061ea0cb9)

![CleanShot 2023-10-03 at 12 36 32@2x](https://github.com/calcom/cal.com/assets/97125783/4be06af8-bd33-4353-a747-3fad1800cfc4)

**Comment by PeerRich** [comment]
can you not use managed event-types for this?

**Comment by its-julian** [comment]
@PeerRich, I just checked but unfortunately no this doesn't work.

1. Managed event types give us the same limiting location options as round-robin.
2. We specifically need round-robin events for our company. Individual managed events aren't useful to us, for our use case.

Hopefully we can add some controls for this into the UI for Admins to control, similar to how Calendly has done it.

**Comment by jeremyccrane** [comment]
Yes, even setting a default **_Location_** under **_Event Setup_** is not working for us with (paid account on Cal.com). We need the system to generate a Google Meet link for booked events.

**Comment by jckw** [comment]
I'm not sure if this is a feature or a bug.

I can go and set the event `locations` on the EventType in the DB manually with:

```
[{"type": "integrations:zoom"}]
```

and then Zoom works completely as expected for team events.

Without it, and with the "Organizer's default app" selected, no Zoom links get created (even when the organiser's default _is_ Zoom).

**Comment by jeesm7** [comment]
I need to be able to select zoom for location for my teams links. 

We host some calls on zooms for our support team because they can use zoom to control the client's screen. 

The rest of our calls are on google meets because its simpler (sales calls mostly)
